packages/core/src/pages/[slug]/p.tsx

```diff
@@ -161,6 +168,8 @@ function Page({ data: server, sections, globalSections, offers, meta }: Props) {
       />
       <BreadcrumbJsonLd itemListElements={itemListElements} />
       <ProductJsonLd
+        id={`${meta.canonical}${settings?.seo?.id ?? ''}`}
```

Comment:
I think we have also to consider the `prefix domain` when adding the storeUrl to generate valid url.
ex:
![image](https://github.com/user-attachments/assets/eaad6d32-5b5d-42a9-b8c1-8f8442cfd191)

https://vtexfaststore.com/office/ vs https://www.vtexfaststore.com/office/

Reply:
Yes! I'll change that for the breadcrumb list also